app/views/taxons/show_a.html.erb

```diff
@@ -2,32 +2,32 @@
 <% content_for :is_taxon_with_subtopics, presented_taxon.show_subtopic_grid? %>
 <%=
   render(
-    partial: 'common',
-    locals: {
-      presented_taxon: presented_taxon
-    }
+      partial: 'common',
```

Comment:
I think this (and a few lines underneath) are indented one tab too far.

Reply:
👍